### PR TITLE
Soarian fhir ascii operation definition write

### PR DESCRIPTION
**Pull request by shiladityadps**
Description
Operation Definition ASCII Doc
----

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.
Foundation_layout
OperationDefinition_1
OperationDefinition_2
OperationDefinition_3
OperationDefinition_4
OperationDefinition_5
OperationDefinition_6
SoarianR4_layout

### CODE REVIEW

File: content/soarian/r4/foundation/conformance/operation-definition.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#### Request
+
+    GET https://fhir-ehr-sc.cerner.com/r4/2f8f5ec1-b7b8-4be5-ae27-e308284dd9c1/OperationDefinition?url=https://fhir-ehr-sc.cerner.com/r4/OperationDefinition/group-export
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:R4_OPERATION_DEFINITION_SEARCH) %>
+
+## Retrieve by id
```
Comment (HRNWriter) [inline content/soarian/r4/foundation/conformance/operation-definition.md:65]: Change to Retrieve by ID.
PRODOC-170481.

**Review by HRNWriter — COMMENTED** [review]
I added a comment requesting a minor change on line 65 of content/soarian/r4/foundation/conformance/operation-definition.md. Otherwise all looks good.

File: content/soarian/r4/foundation/conformance/operation-definition.md (hunk 1 above)
Comment (shiladityadps) [reply content/soarian/r4/foundation/conformance/operation-definition.md]: Changed.

Comment (HRNWriter) [reply content/soarian/r4/foundation/conformance/operation-definition.md]: +1

**Comment by vf056083** [comment]
Approved.

PRODOC-170481

**Review by slvsridhar — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]

**Pull request merged**